### DISCUSSION TITLE
Adding Bokeh support

### DISCUSSION THREAD

**Discussion by bfonta** [scikit-hep/hist#252] [Ideas]
As discussed in the Slack channel of [PyHEP21](https://indico.cern.ch/event/1019958/) as a result of [this tutorial](https://indico.cern.ch/event/1019958/contributions/4420290/), it would be very nice to include [```bokeh```](https://bokeh.org/) as a backend to display plots. Eventually this effort would create a new "```bokehep```" library, just like ```mplhep```.
A different option would be to develop "```hvhep```" (built on top of [HoloViews](https://holoviews.org/), which is much more higher level than ```bokeh```.

**Comment by reckoner** [comment]
Is this on the roadmap? I would be very interested in this.

**Reply by bfonta** [reply]
It is possible I will start working on it next year, but it would help to find other people interested.

**Reply by reckoner** [reply]
[This](https://github.com/PatrikHlobil/Pandas-Bokeh) may be of interest. I would be interested in helping, but I may not have the skills required.
